ci.sh

```diff
@@ -1,18 +1,10 @@
 #!/bin/bash
 set -ex
 
-# Fixed `minirust-tooling` commit, we need to bump this occasionally.
-TOOLING_COMMIT="b1b41a1"
+# Fixed specr-transpile version
+VERSION="0.1.4"
 
-# where to check out the tooling
-TOOLING_DIR="$HOME/minirust-tooling"
+cargo install "specr-transpile@${VERSION}"
+specr-transpile specr.toml
 
-git clone "https://github.com/memoryleak47/minirust-tooling" "$TOOLING_DIR"
-ln -s "$PWD" "$TOOLING_DIR"/minirust
-
-cd "$TOOLING_DIR"
-git checkout "$TOOLING_COMMIT"
-
-# transpile, and build the transpiled result
-(cd specr-transpile; cargo run)
-(cd gen-minirust; RUSTFLAGS="-D warnings" cargo build)
+cd gen-minirust; RUSTFLAGS="-D warnings" cargo build
```

Comment:
```suggestion
(cd gen-minirust; RUSTFLAGS="-D warnings" cargo build)
```
or have it in two lines? not sure why you made this one line